compose.yaml

```diff
@@ -175,16 +175,19 @@ services:
         max-size: "10m"
         max-file: "3"
     environment:
-      CI: "${CI}" # Set in Github actions, changes the Demo TLM_LOG_CYCLE_TIME
```

Comment:
This was always setting the `CI` variable because even though it was a blank string we were just testing for existence. By changing to the list with CI it only sets the variable if the shell has it set.